starboard/player.h

```diff
@@ -532,13 +532,8 @@ SB_EXPORT void SbPlayerWriteSample2(SbPlayer player,
                                     const SbPlayerSampleInfo* sample_infos,
                                     int number_of_sample_infos);
 
-// Writes a single sample of the given media type to |player|'s input stream.
-// Its data may be passed in via more than one buffers.  The lifetime of
-// |sample_buffers|, |sample_buffer_sizes|, |video_sample_info|, and
-// |sample_drm_info| (as well as member |subsample_mapping| contained inside it)
-// are not guaranteed past the call to SbPlayerWriteSample. That means that
-// before returning, the implementation must synchronously copy any information
-// it wants to retain from those structures.
+// Returns the maximum number of samples that can be written in a single call
+// to SbPlayerWriteSamples().
```

Comment:
I am sorry that I didn't provide enough context.  Let's also add a note below like (please rephrase):

(in the same paragraph) Returning a value greater than one allows SbPlayerWriteSamples() to write multiple samples in one call to improve performance.

Note that this feature is currently disabled in Cobalt where SbPlayerWriteSamples() will always be called with one sample.

Reply:
[Follow up PR](https://github.com/youtube/cobalt/pull/2477)